src/main/java/com/slampvp/factory/randomticksystem/RandomTickManager.java

```diff
@@ -0,0 +1,77 @@
+package com.slampvp.factory.randomticksystem;
+
+import com.slampvp.factory.blocks.behaviours.randomtick.RandomTickable;
+import it.unimi.dsi.fastutil.shorts.Short2ObjectMap;
+import it.unimi.dsi.fastutil.shorts.Short2ObjectOpenHashMap;
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.coordinate.Vec;
+import net.minestom.server.event.GlobalEventHandler;
+import net.minestom.server.event.instance.InstanceTickEvent;
+import net.minestom.server.instance.Chunk;
+import net.minestom.server.instance.Instance;
+import net.minestom.server.instance.block.Block;
+
+import java.util.SplittableRandom;
+
+public class RandomTickManager {
+
+    private static final Short2ObjectMap<RandomTickable> randomTickables = new Short2ObjectOpenHashMap<>();
+    private final SplittableRandom random = new SplittableRandom();
```

Comment:
wat is deze random

Reply:
Random met betere performance

Reply:
Ik zou toch gwn threadlocalrandom gebruiken hier
zeker omdat we [split](https://docs.oracle.com/javase/8/docs/api/java/util/SplittableRandom.html#split--)() niet nodig hebben & die niet thread-safe is